types/externs.d.ts

```diff
@@ -170,7 +198,7 @@ declare global {
       chromeIgnoreDefaultFlags: boolean;
       chromeFlags: string | string[];
       /** Output path for the generated results. */
-      outputPath: string;
+      outputPath?: string;
```

Comment:
as far as I can tell, this has always been optional from `cli-flags`, and we rely on at least [the falsy behavior of `outputPath` for report file naming](https://github.com/GoogleChrome/lighthouse/blob/111aaf100c59385c1504d9074820668c0747d2ad/lighthouse-cli/run.js#L139-L141), so seems ok to document the real type coming out of `yargs`